client/app/components/ServicesTemplate/index.js

```diff
@@ -3,27 +3,24 @@ import React from 'react';
 import cn from 'classnames';
 import Card from 'components/SharedComponents/Card';
 import ServicesCardContent from './ServicesCardContent';
-import json from './data.json';
+import data from './data.json';
 
 // Styles
 import styles from './styles.scss';
 
+// TODO - update Card to not take <img> directly, pass src string and maybe sizing/styles as props
 const ServicesTemplate = () => {
-  const filledCardsTop = json.data.topCardContent.map(({ title, content, url, img, id }) =>
```

Comment:
@leseanbiz Notice how this simplifies how the JSON is used on the JavaScript side?